src/Illuminate/Support/Str.php

```diff
@@ -1102,6 +1102,30 @@ public static function random($length = 16)
         })($length);
     }
 
+    /**
+     * Generate a random string containing only lowercase alphabetic characters.
+     *
+     * @param  int  $length
+     * @return string
+     */
+    public static function randomLetters($length = 16)
+    {
+        return (static::$randomStringFactory ?? function ($length) {
+            $characters = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ';
+            $string = '';
+
+            while (($len = strlen($string)) < $length) {
+                $size = $length - $len;
+                $bytesSize = (int) ceil($size / 3) * 3;
+                $bytes = random_bytes($bytesSize);
+
+                $string .= substr(str_replace(['/', '+', '='], '', base64_encode($bytes)), 0, $size);
+            }
+
+            return substr(str_shuffle(str_repeat($characters, ceil($length / strlen($characters)))), 0, $length);
```

Comment:
You could use [`\Random\Randomizer::getBytesFromString`](https://www.php.net/manual/en/random-randomizer.getbytesfromstring.php) for this:
```suggestion
        return (new \Random\Randomizer())
            ->getBytesFromString('abcdefghijklmnopqrstuvwxyz', $length);
```

Reply:
This is great!
But I get the IDE Hint for PHP 8.3 only, as we are on minimum 8.2 stability

<img width="818" alt="Bildschirmfoto 2025-04-21 um 15 08 15" src="https://github.com/user-attachments/assets/7f2eabb1-9cbc-47ed-84a1-92d70109761f" />

Reply:
Ah, damn! Right. Nevermind. This should be refactored in `master` for the `13.x` release then

Reply:
Yes the native bytesFromString is great indeed!
What is the process for marking this as refactoring needed in 13.x?
Anything for me to do?

Reply:
Good question! I'd ask @taylorotwell but I'm not sure if he reads it here. Maybe @crynobone can answer that